src/components/PageWithSideMenu.tsx

```diff
@@ -33,7 +33,7 @@ const StyledSideMenuHeader = styled(StyledPaperHeader)({
 });
 
 export const SidePanel = (props: BoxProps) => (
-  <Box component="section" sx={{ bgcolor: 'secondary.main', minWidth: '20rem' }} {...props} />
+  <Box component="section" sx={{ bgcolor: 'secondary.main', width: { xs: '100%', md: '20rem' } }} {...props} />
```

Comment:
Mener at dette blir bra for både desktop og mobil.
problemet tidligere var at den ble alt for bred om den hadde noe content som var bredt